sparkmagic/setup.py

```diff
@@ -79,16 +79,16 @@ def version(path):
       install_requires=[
           'hdijupyterutils>=0.6',
           'autovizwidget>=0.6',
-          'ipython>=4.0.2,<7',
-          'nose',
-          'mock',
-          'pandas>=0.17.1',
-          'numpy',
-          'requests',
-          'ipykernel>=4.2.2,<5',
-          'ipywidgets>5.0.0,<8.0',
-          'notebook>=4.2,<6.0',
-          'tornado>=4',
-          'requests_kerberos>=0.8.0'
+          'ipython>=5.8.0,<8',
+	  'nose>=1.3.7',
+	  'mock>=3.0.5',
+	  'pandas>=0.24.2',
```

Comment:
same - is this known for `sparkmagic` to be broken on 0.24.1 and earlier versions ?